app/dashboard/views.py

```diff
@@ -2770,7 +2770,7 @@ def get_profile_tab(request, profile, tab, prev_context):
 
                 return TemplateResponse(request, 'profiles/profile_activities.html', context, status=status)
 
-        as_dict = profile.to_dict()
+        as_dict = profile.as_dict
```

Comment:
https://github.com/gitcoinco/web/pull/9075/files#diff-9d08214144399c16f205c599ce310ef6a3edb83fe9b0c47ab7df5c6bbf6b0c3cL2774

instead of computing the dict on the fly, use the saved one in the db.  the saved one in the db is updated by the queues on the backend (not on load time)

cc @gdixon

Reply:
Awesome 🙌 Would it be worth making this same optimisation to kudos?

Reply:
which line is that?